project_usability/README.rst

```diff
@@ -7,7 +7,7 @@ Project Usability Improvements
 ==============================
 
 Several improovements to project:
-#. Add to project issue to search by "Days Since Creation".
+#. Add to project form view visible the fields Dates and Analytic Account.
```

Comment:
Nico, fijate que esto no está muy bien escrito, podría ser algo tipo "Make Dates and Analytic Account fields visible on project form view"